setuper/submissions.py

```diff
@@ -13,3 +13,12 @@ def submission2xml(submission_dict, form_id, form_version_id, gen_uuid=False):
     xml = f'<?xml version=\'1.0\' ?><{root} id="{form_id}" version="{form_version_id}" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms">{inner_xml}</{root}>'
 
     return xml
+
+
+def org_unit_gps_point(org_unit):
+    return {
+        "orgUnitId": org_unit["id"],
```

Comment:
`orgUnitId` is not strictly related to the idea of `gps_point`. Can we live without it here?